lib/package-managers/npm.js

```diff
@@ -85,6 +88,49 @@ function isPre(version) {
     return versionUtil.getPrecision(version) === 'release';
 }
 
+/**
+ * @param {{}}     versions    Object with all versions
+ * @param {String} enginesNode Package engines.node range
+ * @returns {Array} An array of versions which satisfies engines.node range
+ */
+function filterSatisfiesMinVersion(versions, enginesNode) {
+    if (!enginesNode) {
+        return _.keys(versions);
+    }
+    const minVersion = parseMinVersion(enginesNode);
+    if (!minVersion) {
+        return _.keys(versions);
+    }
+    return _.keys(versions).filter((version) => {
+        return isSatisfies(versions[version], minVersion);
+    });
+}
+
+/**
+ * @param {String} enginesNode
+ * @return {Boolean|String} Minimum version of node
+ */
+function parseMinVersion(enginesNode) {
+    const minVersion = semver.minVersion(enginesNode);
+    if (!minVersion || !minVersion.version) {
+        return false;
+    }
+    return minVersion.version;
+}
+
+/**
+ *
+ * @param {{}} version
+ * @param {{}} minVersion
+ * @returns {boolean}
+ */
+function isSatisfies(version, minVersion) {
```

Comment:
Since this function is small and only used once, I would recommend inlining it at the call point.